jest.config.ts

```diff
@@ -27,15 +28,48 @@ export default {
     },
   },
   collectCoverageFrom: [
-    "**/src/**/*.ts",
-    "!**/node_modules/**",
-    "!**/__tests__/**",
-    "!**/src/external-types/**",
-    "!**/src/index.ts",
-    "!**/dist/**",
-    "!**/*.spec.ts",
-    "!**/examples/**",
+    "<rootDir>/src/**/*.ts",
+  ],
+  coveragePathIgnorePatterns: [
+    "node_modules/",
+    "dist/",
+    ".*.spec.ts"
   ],
   injectGlobals: false,
   setupFilesAfterEnv: ['<rootDir>/jest.setup.ts'],
+}
+
+
+export default {
+  projects: [{
+    ...baseConfig,
+    displayName: "core",
+    roots: ["<rootDir>/packages/core"],
+  }, {
+    ...baseConfig,
+    displayName: "oidc",
+    roots: ["<rootDir>/packages/oidc"],
```

Comment:
For clarity, would it be worth renaming this folder to `oidc-browser` (not changing the package name, just the folder name)

Reply:
No strong opinion, but no harm in being more explicit: 1a94ef691cf16b8a33eb8530c406e7e7584979b0

Reply:
Actually this breaks sonar integration, which would create a bit of delay to sort out where this config is. I'd like this PR to get merged now, as it has already lingered quite a while, so let's do the renaming later when I have figured out what needs to be adjusted on sonar.

Reply:
It should just be updating here: https://github.com/inrupt/solid-client-authn-js/blob/main/.github/workflows/ci.yml#L63

Reply:
But I am of course okay doing it separatey

Reply:
Ah cool, it was late yesterday when I was looking into it so I didn't want to investigate, but if that's just that's good.